### PR TITLE
Create science-bridge and cameras ansible scripts

### PR DESCRIPTION
**Pull request by jcp419**
wowee

### CODE REVIEW

File: ansible/roles/jetson_services/files/rover-jetson-cameras.service [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=MRover Cameras
+
+[Service]
+EnvironmentFile=/etc/default/rover.env
+User={{ running_user }}
+ExecStart={{ workspace_dir }}/jarvis exec jetson_cameras_
```
Comment (tabiosg) [inline ansible/roles/jetson_services/files/rover-jetson-cameras.service:7]: jetson_cameras, why the extra underscore :^)

**Review by tabiosg — CHANGES_REQUESTED** [review]
You will also need to change the ansible/roles/jetson_services/tasks/main.yml file

**Comment by tabiosg** [comment]
Never mind, main.yml is already updated (I didn't see newest commit). Anyway, fix the extra underscore

File: ansible/roles/jetson_services/files/rover-jetson-cameras.service (hunk 1 above)
Comment (jcp419) [reply ansible/roles/jetson_services/files/rover-jetson-cameras.service:7]: why must you tear me down

**Pull request closed without merging**